### ISSUE TITLE
host_alert_store.lua error

### ISSUE THREAD

**Issue by rreiner** [ntop/ntopng#8164]
**Environment**:
* OS name: [e.g. Ubuntu]
Raspbian

* OS version: [e.g. 18.04]
10 (Buster)

* Architecture: [e.g. amd64]
armv7l (running on Pi4B but with 32 bit environment)

* ntopng version/revision: [e.g. 5.1.211223]
v.6.1.240108

**What happened**:
Seeing these log entries continually:
```
10/Jan/2024 15:37:26 [LuaEngine.cpp:661] WARNING ...png/scripts/lua/modules/alert_store/host_alert_store.lua:70: bad argument #7 to 'format' (number expected, got nil)
10/Jan/2024 15:37:26 [notifications.lua:25] [host_alert_store.lua:67] alert_category is not set for host alert_id %u
10/Jan/2024 15:10:50 [LuaEngine.cpp:661] WARNING ...png/scripts/lua/modules/alert_store/host_alert_store.lua:70: bad argument #7 to 'format' (number expected, got nil)
10/Jan/2024 15:10:50 [notifications.lua:25] [host_alert_store.lua:67] alert_category is not set for host alert_id %u
10/Jan/2024 14:37:39 [LuaEngine.cpp:661] WARNING ...png/scripts/lua/modules/alert_store/host_alert_store.lua:70: bad argument #7 to 'format' (number expected, got nil)
10/Jan/2024 14:37:39 [notifications.lua:25] [host_alert_store.lua:67] alert_category is not set for host alert_id %u
10/Jan/2024 13:45:59 [LuaEngine.cpp:661] WARNING ...png/scripts/lua/modules/alert_store/host_alert_store.lua:70: bad argument #7 to 'format' (number expected, got nil)
```

Also receiving similar messages as alerts:
```
NtopngNotifications
APP  [1:46 PM](https://reiner.slack.com/archives/C06D9QV7342/p1704912365126639)
[10/01/2024 13:46:01]  [System] [System Error] [system]  System error detected in ntopng, please report it. Error message: ...png/scripts/lua/modules/alert_store/host_alert_store.lua:70: bad argument #7 to 'format' (number expected, got nil)
NtopngNotifications
APP  [2:38 PM](https://reiner.slack.com/archives/C06D9QV7342/p1704915482631209)
[10/01/2024 14:38:01]  [System] [System Error] [system]  System error detected in ntopng, please report it. Error message: ...png/scripts/lua/modules/alert_store/host_alert_store.lua:70: bad argument #7 to 'format' (number expected, got nil)
NtopngNotifications
APP  [3:11 PM](https://reiner.slack.com/archives/C06D9QV7342/p1704917463748019)
[10/01/2024 15:11:02]  [System] [System Error] [system]  System error detected in ntopng, please report it. Error message: ...png/scripts/lua/modules/alert_store/host_alert_store.lua:70: bad argument #7 to 'format' (number expected, got nil)
NtopngNotifications
APP  [3:38 PM](https://reiner.slack.com/archives/C06D9QV7342/p1704919085989159)
[10/01/2024 15:38:01]  [System] [System Error] [system]  System error detected in ntopng, please report it. Error message: ...png/scripts/lua/modules/alert_store/host_alert_store.lua:70: bad argument #7 to 'format' (number expected, got nil)
```
**How did you reproduce it?**
Running the system normally, seeing these all the time

**Comment by cardigliano** [comment]
I added some info to the trace, a new update will be available later today. Please provide the new trace after the update. Thank you

**Comment by paulgiordano** [comment]
We are also experiencing the issue. what is the update you'd like pulled to get more info? 
OS Ubuntu 22.04.3
ntopng v.6.1.240109 
architecture x86_64

**Comment by NicoMaio** [comment]
Fixed a similar issue on #8204. Please update and test again.

**Comment by MatteoBiscosi** [comment]
Verified as fixed, please reopen if needed
